dashboard/db/migrate/20170306221907_merge_professional_learning_partner_into_regional_partner.rb

```diff
@@ -0,0 +1,34 @@
+class MergeProfessionalLearningPartnerIntoRegionalPartner < ActiveRecord::Migration[5.0]
+  def up
+    ActiveRecord::Base.record_timestamps = false
+    begin
+      ProfessionalLearningPartner.all.each do |plp|
+        regional_partner = RegionalPartner.find_or_create_by!(name: plp.name)
+        regional_partner.contact_id = plp.contact_id
+        regional_partner.urban = plp.urban
+        regional_partner.save!
+      end
+    ensure
+      ActiveRecord::Base.record_timestamps = true
+    end
+
+    drop_table :professional_learning_partners
```

Comment:
It looks like you'll be ok if you upgrade those 2 references to `ProfessionalLearningPartner` to use `RegionalPartner` instead.